packages/react-test-renderer/shallow.js

```diff
@@ -9,8 +9,4 @@
 
 'use strict';
 
-const ReactShallowRenderer = require('./src/ReactShallowRenderer');
-
-// TODO: decide on the top-level export form.
-// This is hacky but makes it work with both Rollup and Jest.
-module.exports = ReactShallowRenderer.default || ReactShallowRenderer;
+export {default} from 'react-shallow-renderer';
```

Comment:
So what does this do? Is it using an actual ES build? Is this relying on the `module` field? @sebmarkbage Didn't you have concern about doing this?

Reply:
https://gist.github.com/NMinhNguyen/9d6863f70b44b953092c842465b3be90

Reply:
Actually, I added `console.log({ importee, importer })` in the Rollup forks plugin and seems like it picked ESM :)

```
{
  importee: '/react/packages/react-test-renderer/shallow.js',
  importer: undefined
}
{
  importee: '/react/packages/react-test-renderer/shallow.js',
  importer: undefined
}
{
  importee: '/react/packages/react-test-renderer/shallow.js',
  importer: undefined
}
{
  importee: 'react-shallow-renderer',
  importer: '/react/packages/react-test-renderer/shallow.js'
}
{
  importee: './react-shallow-renderer',
  importer: '/react/node_modules/react-shallow-renderer/esm/index.js'
}
{
  importee: 'object-assign',
  importer: '/react/node_modules/react-shallow-renderer/esm/react-shallow-renderer.js'
}
{
  importee: 'react-is',
  importer: '/react/node_modules/react-shallow-renderer/esm/react-shallow-renderer.js'
}
{
  importee: 'prop-types/checkPropTypes',
  importer: '/react/node_modules/react-shallow-renderer/esm/react-shallow-renderer.js'
}
{
  importee: './lib/ReactPropTypesSecret',
  importer: '/react/node_modules/prop-types/checkPropTypes.js'
}
{
  importee: '\u0000commonjs-proxy-./lib/ReactPropTypesSecret',
  importer: '/react/node_modules/prop-types/checkPropTypes.js'
}
{
  importee: './lib/ReactPropTypesSecret',
  importer: '/react/node_modules/prop-types/checkPropTypes.js'
}
{
  importee: './src/ReactIs',
  importer: '/react/packages/react-is/index.js'
}
{
  importee: '/react/node_modules/prop-types/lib/ReactPropTypesSecret.js',
  importer: '\u0000commonjs-proxy-/react/node_modules/prop-types/lib/ReactPropTypesSecret.js'
}
{
  importee: 'shared/ReactSymbols',
  importer: '/react/packages/react-is/src/ReactIs.js'
}
{
  importee: 'shared/isValidElementType',
  importer: '/react/packages/react-is/src/ReactIs.js'
}
{
  importee: 'shared/ReactSymbols',
  importer: '/react/packages/shared/isValidElementType.js'
```